src/Resources/translations/messages.en.yaml

```diff
@@ -21,6 +21,12 @@ monsieurbiz_menu:
         help_noopener: 'Add "noopener" to the rel attribute'
         nofollow: 'nofollow'
         help_nofollow: 'Add "nofollow" to the rel attribute'
+        choose_object: 'Choose an object'
```

Comment:
Url Autocomplete is not correct because it's not an autocomplete
Same for Autocomplete helper.

I think `Choose an object` is simple to understand for non tech users.

Reply:
Agree to disagree.
Maybe URL helper ? :P

Reply:
Changed to `Choose a target`